.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
This lil red stop sign is GitHub's way of complaining that there's no newline at the end of the file. Always putting a newline at the end of every file is a good habit to get into because it can minimize the risk of file parsing / processing snafus on *nix machines. Here's one good discussion: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Remember to add this new line over here, either manually or by installing the VSCode extension Frank mentioned in our groupchat.